Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.h

```diff
@@ -225,12 +225,18 @@ class TetrahedronFEMForceField : public core::behavior::ForceField<DataTypes>, p
     Data<type::vector<Real> > _vonMisesPerElement; ///< von Mises Stress per element
     Data<type::vector<Real> > _vonMisesPerNode; ///< von Mises Stress per node
     Data<type::vector<type::Vec4f> > _vonMisesStressColors; ///< Vector of colors describing the VonMises stress
-    
+
+    Real _minVMN;
+    Real _maxVMN;
+
     Data<std::string> _showStressColorMap; ///< Color map used to show stress values
     Data<float> _showStressAlpha; ///< Alpha for vonMises visualisation
     Data<bool> _showVonMisesStressPerNode; ///< draw points showing vonMises stress interpolated in nodes
+    Data<bool> d_showVonMisesStressPerNodeColorMap; ///< draw triangles showing vonMises stress interpolated in nodes
     Data<bool> _showVonMisesStressPerElement; ///< draw triangles showing vonMises stress interpolated in elements
 
+    Data<float> d_showElementGapScale; ///< draw gap between elements (when showWireFrame is disabled)
```

Comment:
Considering the computations where `d_showElementGapScale` is involved, I would set its type to `Real`
```suggestion
    Data<Real> d_showElementGapScale; ///< draw gap between elements (when showWireFrame is disabled)
```